### PR TITLE
Add data model and queue

### PR DESCRIPTION
**Pull request by tank157**
Creating PR, documentation still in progress, will assign for review and acceptance when that is complete.

### CODE REVIEW

**Comment by tank157** [comment]
[Torchpad Documentation](https://torchpad.com/workspace/wikis/democracyworks/pages/kraken/election-works)

[Pivotal Card 1](https://www.pivotaltracker.com/story/show/91998840)
[Pivotal Card 2](https://www.pivotaltracker.com/story/show/93450256)

**Comment by tank157** [comment]
Assigning to @tie-rack , since we can go over this as a pair, and he was also involved in the similar user-works, errr, work.

File: src/election_works/handlers.clj [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+               :message (.getMessage semantic-error)}})
+    (catch clojure.lang.ExceptionInfo validation-error
+      {:status :error
+       :error {:type :validation
+               :message (.getMessage validation-error)}})
+    (catch RuntimeException runtime-exception
+      (log/error runtime-exception)
+      {:status :error
+       :error {:type :server
+               :server (str "Unknown server error: " (.getMessage runtime-exception))}})))
```
Comment (tie-rack) [inline src/election_works/handlers.clj]: All this copied error handling looks like it could be extracted to a function.

Comment (tank157) [inline src/election_works/handlers.clj]: Yeah, I've been pondering that as well. Let me see what I can cook up quickly.

**Comment by tie-rack** [comment]
Great. LGTM.

**Pull request merged**